coingecko/errors.ts

```diff
@@ -1,25 +0,0 @@
-export enum ErrorType {
-  InvalidSearchParams = 10001,
-  InvalidSymbol = 10002,
-  InvalidCoingeckoResponse = 10003,
-  InvalidResponseTypes = 10004,
-}
-
-export const invalidSymbolErrorResponse = (symbol: string[]) => {
-  return {
-    code: ErrorType.InvalidSymbol,
-    message: `${symbol.join(", ")} not exist or not supported`,
-  };
-};
```

Comment:
also, to make debugging a breeze consider using custom errors instead of this beauties

Reply:
don't understand what exactly you wanted to change